src/ts/components/seekbar.ts

```diff
@@ -848,6 +848,7 @@ export class SeekBar extends Component<SeekBarConfig> {
     // value calculated from the size. This required size is not known when it is hidden.
     // For such cases, we refresh the position here in onShow because here it is guaranteed that the component knows
     // its size and can set the position correctly.
-    this.refreshPlaybackPosition();
+    // Some css component use an animation to setup, to make sure it's finished we add a timeout of 300ms which correspond to the 0.3s value (see $animation-duration in _variable.scss)
+    setTimeout( this.refreshPlaybackPosition, 300);
```

Comment:
This is a hack that can't make it into the code I'm afraid. We cannot hardcode things like the animation time that are variable and customizable by definition, as all CSS properties are. I recognize the issue here, but we need to find a proper solution.

Reply:
We could try to not refresh the playback position then in the case if the volume slider ( or the legacy ui) since this is only really useful when switching from and to livestream.

But this can't be done by implementing a `onShowEvent` inside volumeslider since it still needs to to call `super.onShowEvent()`.

Reply:
i guess the seekbar and the volume slider should implement a generic `slider`, or we could reuse the `configureSeek` flag from the `configure` method (making it a class property probably).

Reply:
Yes there should be a common `Slider` base class, the volume bar in it's current state is kind of a hack too.

If leaving out the `refreshPlaybackPosition` from `onShowEvent` works, you can solve it in `VolumeSlider` by overriding `refreshPlaybackPosition`:

```ts
protected refreshPlaybackPosition(refresh: boolean = false) {
  if (refresh) {
    super.refreshPlaybackPosition();
  }
}
```

And then change all `refreshPlaybackPosition()` calls in `VolumeSlider` to `refreshPlaybackPosition(true)`. The call from `onShowEvent` will then default to `refreshPlaybackPosition(true)` and not refresh the position.

Reply:
Yeah that should work, although this feel even more hacky.